app/src/main/java/com/orange/ouds/app/ui/tokens/TokenProperty.kt

```diff
@@ -140,6 +146,30 @@ sealed class TokenProperty<T>(
         categoryClass = TokenCategory.Dimension.Size::class
     )
 
+    data object SizeMaxWidth : TokenProperty<TokenCategory.Dimension.Size>(
+        nameRes = R.string.app_tokens_dimension_size_maxWidth_label,
+        tokens = listOf(
+            OudsSizes.MaxWidth.Type.Display::class,
+            OudsSizes.MaxWidth.Type.Heading::class,
+            OudsSizes.MaxWidth.Type.Body::class,
+            OudsSizes.MaxWidth.Type.Label::class,
+        ).flatMap { getTokens(it) },
```

Comment:
If I am correct, this could be replaced by a single line:

```kotlin
tokens = getTokens<OudsSizes.MaxWidth>(),
```

The difference is that the order is different, but I think it's OK.